cool-demos/slash-commands.html

```diff
@@ -0,0 +1,225 @@
+<!--
+    This demo showcases how to use the autocompleter plugin to make a
```

Comment:
```suggestion
    This demo showcases how to use the autocompleter API to make a
```
It's not a plugin so we probably shouldn't mention that 🤷‍♀️